src/ORM/README.md

```diff
@@ -28,7 +28,8 @@ ConnectionManager::config('default', [
 	'driver' => 'Cake\Database\Driver\Mysql',
 	'database' => 'test',
 	'username' => 'root',
-	'password' => 'secret'
+	'password' => 'secret',
+	'cacheMetaData' => false // If set to `true` you need to install the optional "cakephp/cache" package.
```

Comment:
Why don't we install it in any way?
Also install means composer require and plugin load?